companies.txt

```diff
@@ -1,133 +1,202 @@
-google
-amazon
-facebook
-microsoft
-bloomberg
+accolite
```

Comment:
How did you updates the companies.txt? Was there a script you used or did you do it manually?